crates/red_knot_python_semantic/src/types.rs

```diff
@@ -195,6 +195,8 @@ pub enum Type<'db> {
     LiteralString,
     /// A bytes literal
     BytesLiteral(BytesLiteralType<'db>),
+    /// A heterogeneous tuple type, with elements of the given types in source order.
+    Tuple(TupleType<'db>),
```

Comment:
Maybe we should use `TupleLiteral` instead? This is because there could also be `tuple[int, str]` or `tuple[int, ...]`. Later, we could merge them under the same enum and have them as a single variant in `Type::Tuple`.

Reply:
`TupleLiteral` sounds reasonable to me!